shared/src/androidMain/kotlin/com/fbada006/shared/utils/ExportHelper.android.kt

```diff
@@ -0,0 +1,68 @@
+package com.fbada006.shared.utils
+
+import android.content.Context
+import android.content.Intent
+import android.content.Intent.createChooser
+import android.graphics.Bitmap
+import android.media.MediaScannerConnection
+import android.net.Uri
+import android.os.Environment
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asAndroidBitmap
+import androidx.core.content.ContextCompat.startActivity
+import io.fbada006.artmaker.Res
+import io.fbada006.artmaker.share_your_image
+import kotlinx.coroutines.suspendCancellableCoroutine
+import java.io.File
+import kotlin.coroutines.resume
+
+private suspend fun scanFilePath(context: Context, filePath: String): Uri? = suspendCancellableCoroutine { continuation ->
+    MediaScannerConnection.scanFile(
+        context,
+        arrayOf(filePath),
+        arrayOf("image/png"),
+    ) { _, scannedUri ->
+        if (scannedUri == null) {
+            continuation.cancel(
+                Exception(
+                    "File $filePath could not be scanned",
+                ),
+            )
+        } else {
+            continuation.resume(scannedUri)
+        }
+    }
+}
+
+suspend fun getUriFromBitmap(context: Context, bitmap: ImageBitmap?): Uri {
```

Comment:
This function is same as saveToDisk from the ExportUtils. Since we are interested with the Uri I think this function name makes more sense than the previous name